test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -677,19 +685,33 @@ fi
 
 if [ "$patch_version" = true ]; then
     for b in ${patch_version_upgrade_downgrade_release_branches[@]}; do
-        # Build the tip of the release branch and run test to genearte data files
-        (build_branch $b)
+        # Build the tip of the release branch and run test to generate data files
+        (build_branch $b 1)
         (run_test_checkpoint "$b" "row")
 
         # Pick a patch version from the list of patch versions for the release branch
-        cd $b; pv=$(pick_a_version $b); cd ..
-        (build_branch $pv)
+        cd $b; pv=$(pick_a_version $b); echo "$pv"; cd ..
+
+        (build_branch $pv 1)
+        rtn=$(is_test_checkpoint_recovery_supported $pv)
+
+        # Apply patch fix from WT-8708 to already released compatible versions to avoid test/checkpoint setting commit timestamp less than stable timestamp
+        if [ "$pv" \< "mongodb-4.4.13" ] || [ "$pv" \< "mongodb-5.0.7" ] && [ $rtn == "yes" ]; then
```

Comment:
According to the release schedule, WT-8708 is likely to be included in mongo 5.0.8 and 4.4.14/4.4.15.

Reply:
Is it possible to search git history to see if WT-8708 is part of the given release branch (e.g. mongodb-4.4)?

Reply:
Yep, will work on a change that removes the check against version numbers and instead searches the git logs to see if WT-8708 is present and if not apply the patch for it. Thanks for the suggestion!